site/content/docs/user/quick-start.md

```diff
@@ -377,6 +377,32 @@ networking:
   ipFamily: ipv6
 ```
 
+#### Dual Stack clusters
+You can run dual stack clusters using `kind`, on kubernetes versions +1.16
+but first you need to [enable ipv6 in your docker daemon][docker enable ipv6].
+
+```yaml
+# a dual-stack cluster
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+networking:
+  ipFamily: DualStack
+nodes:
+- role: control-plane
+- role: worker
+- role: worker
+featureGates:
+  IPv6DualStack: true
+kubeadmConfigPatches:
+- |
+  apiVersion: kubeadm.k8s.io/v1beta2
+  kind: ClusterConfiguration
+  metadata:
+    name: config
+  featureGates:
+    IPv6DualStack: true
```

Comment:
drop this patch

Reply:
kubeadm featuregates follow a different path, still needed

Reply:
but I think we can remove it